source/_partials/testimonial_card.blade.php

```diff
@@ -0,0 +1,42 @@
+<section class="section_testimonial">
+    <p class="title-testimonial text-center mt-3">{{ $page->t('Testimonials') }}</p>
+    <h2 class="text-center mb-5">{{ $page->t('What our customers says') }}</h2>
+    <button class="pre-btn"><img src="{{ $page->baseUrl }}assets/images/arrow.png" alt="back_comments"></button>
```

Comment:
The alt text need to be a translatable text. A screen reader will read back_comments. Maybe would be best:
Go back pagination of testimonials.